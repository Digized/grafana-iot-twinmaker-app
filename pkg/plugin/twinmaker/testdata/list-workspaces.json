{
    "NextToken": null,
    "WorkspaceSummaries": [
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/AlarmWorkspace",
            "CreationDateTime": "2022-04-27T17:26:14.482Z",
            "Description": "",
            "UpdateDateTime": "2022-04-27T17:26:14.482Z",
            "WorkspaceId": "AlarmWorkspace"
        },
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-29",
            "CreationDateTime": "2021-11-29T20:48:11.816Z",
            "Description": "",
            "UpdateDateTime": "2021-11-29T20:48:11.816Z",
            "WorkspaceId": "CookieFactory-11-29"
        },
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/GrafanaWorkspace",
            "CreationDateTime": "2021-12-15T00:03:55.82Z",
            "Description": "",
            "UpdateDateTime": "2021-12-15T00:03:55.82Z",
            "WorkspaceId": "GrafanaWorkspace"
        },
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/TestWorkspace",
            "CreationDateTime": "2021-12-06T22:45:50.526Z",
            "Description": "",
            "UpdateDateTime": "2021-12-06T22:45:50.526Z",
            "WorkspaceId": "TestWorkspace"
        },
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/TwinMakerWorkspace",
            "CreationDateTime": "2022-04-07T18:53:04.114Z",
            "Description": "",
            "UpdateDateTime": "2022-04-07T18:53:04.114Z",
            "WorkspaceId": "TwinMakerWorkspace"
        }
    ]
}