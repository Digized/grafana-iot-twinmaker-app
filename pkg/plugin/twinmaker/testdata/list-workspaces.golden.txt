ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: 
Dimensions: 4 Fields by 5 Rows
+---------------------------------------------------------------------------+-----------------------------------+-------------------+---------------------+
| Name: arn                                                                 | Name: created                     | Name: description | Name: workspaceId   |
| Labels:                                                                   | Labels:                           | Labels:           | Labels:             |
| Type: []*string                                                           | Type: []time.Time                 | Type: []*string   | Type: []*string     |
+---------------------------------------------------------------------------+-----------------------------------+-------------------+---------------------+
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/AlarmWorkspace      | 2022-04-27 17:26:14.482 +0000 UTC |                   | AlarmWorkspace      |
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-29 | 2021-11-29 20:48:11.816 +0000 UTC |                   | CookieFactory-11-29 |
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/GrafanaWorkspace    | 2021-12-15 00:03:55.82 +0000 UTC  |                   | GrafanaWorkspace    |
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/TestWorkspace       | 2021-12-06 22:45:50.526 +0000 UTC |                   | TestWorkspace       |
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/TwinMakerWorkspace  | 2022-04-07 18:53:04.114 +0000 UTC |                   | TwinMakerWorkspace  |
+---------------------------------------------------------------------------+-----------------------------------+-------------------+---------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////cAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAFP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA0/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFT+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFwBAADgAAAAaAAAAAQAAADG/v//FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAALT+//8IAAAAFAAAAAsAAAB3b3Jrc3BhY2VJZAAEAAAAbmFtZQAAAAAAAAAAtP7//wsAAAB3b3Jrc3BhY2VJZAAm////FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAABT///8IAAAAFAAAAAsAAABkZXNjcmlwdGlvbgAEAAAAbmFtZQAAAAAAAAAAFP///wsAAABkZXNjcmlwdGlvbgAAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAPAAAAEQAAAAAAAAKRAAAAAEAAAAEAAAAiP///wgAAAAQAAAABwAAAGNyZWF0ZWQABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAHAAAAY3JlYXRlZAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAQAAAAEQAAAAAAAUBQAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAAAwAAAADAAAAYXJuAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAMAAABhcm4AAAAAAP////9IAQAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAIAIAAAAAAAAUAAAAAAAAAwQACgAYAAwACAAEAAoAAAAUAAAAyAAAAAUAAAAAAAAAAAAAAAsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYAAAAAAAAABgAAAAAAAAAXgEAAAAAAAB4AQAAAAAAAAAAAAAAAAAAeAEAAAAAAAAoAAAAAAAAAKABAAAAAAAAAAAAAAAAAACgAQAAAAAAABgAAAAAAAAAuAEAAAAAAAAAAAAAAAAAALgBAAAAAAAAAAAAAAAAAAC4AQAAAAAAABgAAAAAAAAA0AEAAAAAAABQAAAAAAAAAAAAAAAEAAAABQAAAAAAAAAAAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAAFAAAAAAAAAAAAAAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAABEAAAAjQAAANMAAAAWAQAAXgEAAGFybjphd3M6aW90dHdpbm1ha2VyOnVzLWVhc3QtMToxNjY4MDA3NjkxNzk6d29ya3NwYWNlL0FsYXJtV29ya3NwYWNlYXJuOmF3czppb3R0d2lubWFrZXI6dXMtZWFzdC0xOjE2NjgwMDc2OTE3OTp3b3Jrc3BhY2UvQ29va2llRmFjdG9yeS0xMS0yOWFybjphd3M6aW90dHdpbm1ha2VyOnVzLWVhc3QtMToxNjY4MDA3NjkxNzk6d29ya3NwYWNlL0dyYWZhbmFXb3Jrc3BhY2Vhcm46YXdzOmlvdHR3aW5tYWtlcjp1cy1lYXN0LTE6MTY2ODAwNzY5MTc5OndvcmtzcGFjZS9UZXN0V29ya3NwYWNlYXJuOmF3czppb3R0d2lubWFrZXI6dXMtZWFzdC0xOjE2NjgwMDc2OTE3OTp3b3Jrc3BhY2UvVHdpbk1ha2VyV29ya3NwYWNlAACAWJ5Q2tDpFgDaBgdnH7wWAKPX58nEwBaAyxZ84ku+FoAgtuz7seMWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA4AAAAhAAAAMQAAAD4AAABQAAAAQWxhcm1Xb3Jrc3BhY2VDb29raWVGYWN0b3J5LTExLTI5R3JhZmFuYVdvcmtzcGFjZVRlc3RXb3Jrc3BhY2VUd2luTWFrZXJXb3Jrc3BhY2UQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAABAABAAAAgAIAAAAAAABQAQAAAAAAACACAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAFP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA0/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFT+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFwBAADgAAAAaAAAAAQAAADG/v//FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAALT+//8IAAAAFAAAAAsAAAB3b3Jrc3BhY2VJZAAEAAAAbmFtZQAAAAAAAAAAtP7//wsAAAB3b3Jrc3BhY2VJZAAm////FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAABT///8IAAAAFAAAAAsAAABkZXNjcmlwdGlvbgAEAAAAbmFtZQAAAAAAAAAAFP///wsAAABkZXNjcmlwdGlvbgAAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAPAAAAEQAAAAAAAAKRAAAAAEAAAAEAAAAiP///wgAAAAQAAAABwAAAGNyZWF0ZWQABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAHAAAAY3JlYXRlZAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAQAAAAEQAAAAAAAUBQAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAAAwAAAADAAAAYXJuAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAMAAABhcm4AmAIAAEFSUk9XMQ==
